content/community/index.page.tsx

```diff
@@ -1,528 +1,648 @@
-                    className="w-100"
-                  />
-                  <h6 className="mt-3">{translate("XRPL Hackathon 2022")}</h6>
-                </div>
-              </div>
-              <div>
-                <a
-                  className="btn btn-primary btn-arrow"
-                  target="_blank"
-                  href="/events.html"
-                >
-                  {translate("View All Events")}
-                </a>
-              </div>
+              <a
+                href="https://xrpl.org/blog/2023/xrpcafe.html"
+                target="_blank"
+                className="view-project external-link"
+              >
+                {translate("View Project")}
+              </a>
             </div>
           </div>
-        </section>
-        
-        {/* Careers */}
-        <section className="container-new" id="xrpl-careers">
-          <div className="card-grid card-grid-2xN">
-            <div id="careersDark" className="col pr-2 d-lg-block d-none">{CareersDarkLg}</div>
-            <div className="col pt-5">
-              <div className="d-flex flex-column-reverse mb-lg-2 pl-0">
-                <h2 className="h4 h2-sm">
-                  {translate(
-                    "Discover your next career opportunity in the XRPL community"
-                  )}
-                </h2>
-                <h6 className="eyebrow mb-3">{translate("XRPL Careers")}</h6>
-              </div>
-              <p className="mb-3 py-4">
-                {translate(
-                  "Teams across the XRPL community are looking for talented individuals to help build their next innovation."
-                )}
-              </p>
-              <div id="careersDark-small" className="d-lg-none d-block">{CareersDarkSm}</div>
-              <div className="d-lg-block">
-                <a
-                  className="btn btn-primary btn-arrow"
-                  target="_blank"
-                  href="https://jobs.xrpl.org/jobs"
-                >
-                  {translate("View Open Roles")}
-                </a>
-              </div>
+        </div>
+      </section>
+      {/* Bottom Cards Section */}
+      <section className="bottom-cards-section">
+        <div className="com-card">
+          <img className="top-left-img" alt="Top Left Image" />
+          <div className="card-content">
+            <h6 className="card-title">
+              {translate("Contribute to Consensus")}
+            </h6>
+            <h6 className="card-subtitle">
+              {translate("Run an XRP Ledger network node")}
+            </h6>
+            <p className="card-description">
+              {translate(
+                "Thank you for your interest in contributing to XRPL.org."
+              )}
+            </p>
+            <div className="card-links">
+              <a
+                className="com-card-link"
+                target="_blank"
+                href="/the-rippled-server"
```

Comment:
Here's a path that'll actually work (for now; it'll have to change again eventually)

```suggestion
                href="/concepts/networks-and-servers/"
```